### DISCUSSION TITLE
Multiple Paths

### DISCUSSION THREAD

**Discussion by bbgun75** [SixLabors/ImageSharp.Web#328] [Q&A]
Can you specify multiple paths to read from?

Here is our current code.

            services.AddImageSharp()
                .Configure<PhysicalFileSystemCacheOptions>(options =>
                {
                    options.CacheFolder = "ImageSharpCache";
                })
                .ClearProviders()
                .AddProvider<CustomPhysicalFileSystemProvider>()
                .Configure<CustomPhysicalFileSystemProviderOptions>(options =>
                {
                   
                    // How do we make it work with this multisite?
                    options.ContentPath = Path.Combine(Directory.GetCurrentDirectory(), @"CONEXPO_CON_AGG", @"media");
                    //options.ContentPath = Path.Combine(Directory.GetCurrentDirectory(), @"TheUtilityExpo", @"media");
                    options.RequestPath = "/CONEXPO_CON_AGG/media";
                    //options.RequestPath = "/TheUtilityExpo/media";
                })
            .AddProvider<PhysicalFileSystemProvider>();

**Comment by JimBobSquarePants** [comment]
You do the logic in your `IImageProvider` implementation and design the URLs (generally using different pathbase values) to work with that logic.
